docs/getting-started.md

```diff
@@ -0,0 +1,144 @@
+# Getting Started - BRK Bevragen API
+De 'BRK Bevragen' API is gespecificeerd met behulp van de OpenAPI Specifications (OAS3).
+
+Om aan te sluiten kunt u de volgende stappen doorlopen:
+1. [Meld je aan bij het kadaster om toegang te krijgen](#Aanmelden-om-aan-te-sluiten)
+2. [Bekijk de functionaliteit en specificaties](#Functionaliteit)
+3. [Probeer en test de API](#Probeer-en-test-de-API)
+4. [Sluit aan op productie](#Aansluiten-op-productie)
+
+## Aanmelden om aan te sluiten
+Meld je aan bij het kadaster om [aan te sluiten](https://formulieren.kadaster.nl/aanmelden_brk_bevragen) en voor [toegang tot de testomgeving](https://formulieren.kadaster.nl/aanmelden_brk_bevragen).
```

Comment:
Dit zijn allebei dezelfde links. Als er maar 1 link is dan misschien in je tekst ook aangeven dat er 1 plek is waar je je zowel voor de productie-omgeving als voor de testomgeving moet aanmelden.

Reply:
opgelost door beide links samen te voegen tot één link